### PR TITLE
added question and answer of codeforces 676

### PR DESCRIPTION
**Pull request by noob3426**

### CODE REVIEW

File: Task 1/XORwice/solution.cpp [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            ll temp3=temp2 / temp;
+
+            count +=temp;
+
+
+
+       }
+       
+
+
```
Comment (chandrikadeb7) [inline Task 1/XORwice/solution.cpp:91]: Remove the whitespaces please.

**Review by noob3426 — COMMENTED** [review]
removed whitespaces

File: Task 1/XORwice/solution.cpp (hunk 1 above)
Comment (noob3426) [reply Task 1/XORwice/solution.cpp]: can you please review it asap?

**Review by nimishbongale — APPROVED** [review]
Great job @noob3426 @ak0982 ! 🚀 

You are now all set to contribute to cool Open Source projects!

**Pull request merged**